### PR TITLE
Add steps to update performance test clusters when upgrading k8s version

### PR DESCRIPTION
**Pull request by gongmax**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup

 /kind documentation

> /kind feature
> /kind hotfix
> /kind release

**What this PR does / Why we need it**:
 We introduced a performance test cluster recently. It's version needs to be updated when we update the supported k8s version. Add steps for the purpose.

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/kubernetes_update.md [hunk 1]
```diff
@@ -46,12 +46,21 @@ List of items to do for upgrading to {version_1} {version_2} {version_3}
...
     - [ ] Update the Cloud Build configuration to run e2e test on the new created clusters, and disable the e2e test on the cluster with the oldest supported K8s version
         - [ ] Update the `versionsAndRegions` variable to add the new supported version and remove the oldest supported K8s version in `cloudbuild.yaml` `submit-e2e-test-cloud-build` step
         - [ ] Run `make lint` for code quality check.
         - [ ] Submit a PR to trigger the e2e tests and verfiy they all pass
     - [ ] After the PR that includes the above Cloud Build configuration change has been merged and all the existing pending PRs in the Cloud Build queue have picked up the new configuration, submit a separate PR to update the e2e clusters terraform module to remove the e2e cluster with the oldest supported K8s version.
-        - [ ] In `terraform/e2e/module.tf`, remove the oldest supported version from the map `kubernetes_versions`.
+        - [ ] In `build/terraform/e2e/module.tf`, remove the oldest supported version from the map `kubernetes_versions`.
         - [ ] Destroy the old clusters with new scripts: `cd build; make GCP_PROJECT=agones-images gcloud-e2e-test-cluster`
```
Comment (Kalaiselvi84) [inline .github/ISSUE_TEMPLATE/kubernetes_update.md:57]: If I understand correctly, the `make GCP_PROJECT=agones-images gcloud-e2e-test-cluster` command didn't remove the cluster last time, so we used `terraform init -backend-config="bucket=agones-images-e2e-infra-bucket-tfstate" -backend-config="prefix=terraform/state"
terraform apply -var project="agones-images"` command. Was this an intermittent issue, or do we need to update it?

Comment (gongmax) [reply .github/ISSUE_TEMPLATE/kubernetes_update.md:57]: I have a fix for that. https://github.com/googleforgames/agones/pull/3468 still pending review. No need to update the docs here.

Comment (Kalaiselvi84) [reply .github/ISSUE_TEMPLATE/kubernetes_update.md:57]: Cool, all good!

**Comment by Kalaiselvi84** [comment]
LGTM!

**Review by markmandel — APPROVED** [review]

**Pull request merged**